api/db/services/tenant_llm_service.py

```
@@ -556,12 +556,20 @@ def close(self):
-        # Drop the Langfuse reference WITHOUT flushing/shutting down the shared
-        # client (see the docstring above for why this would deadlock).
-        self.langfuse = None
+        # Flush and shutdown Langfuse client if it was initialized
+        if self.langfuse:
+            try:
+                self.langfuse.flush()
+                if hasattr(self.langfuse, "shutdown"):
+                    self.langfuse.shutdown()
+            except Exception:
+                # Ignore errors during cleanup
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)